### ISSUE TITLE
Make getMimeType function configurable

### ISSUE THREAD

**Issue by maxlysenko** [getgrav/grav#369]
I suggest to create some configuration for mime types.
For example, add something like this
```yaml
mimetypes:
  - mimetype: 'image/jpg'
    extensions: [jpeg, jpg, jpe]
  - mimetype: 'image/svg+xml'
    extensions: [svg]
```
into `system.yaml` or into separate `mimetypes.yaml` configuration file.

Then add something like this
```php
$mimetypes = self::getGrav()['config']->get('mimetypes.mimetypes', []);

foreach ($mimetypes as $mimetype) {
    if (in_array($extension, $mimetype['extensions'])) {
        return $mimetype['mimetype'];
    }
}
```
into the `getMimeType` function in the `Utils.php`.

In that way it will be possible to add some extensions/mime types/content types without altering source code. So if it is needed to add `xlsx`, `pptx`, or some other extension or content type it could be done it the configuration file inside `user/config` folder.

**Comment by rhukster** [comment]
It is already configurable via the `system/config/media.yaml` file.  Simply copy this file or add your own in a `user/config/media.yaml` and they will be merged like any other Grav config.

**Comment by maxlysenko** [comment]
You know, SVG file was described in the `system/config/media.yaml`.
So why SVG was not working until @flaviocopes fixed #366 .
And where is it already configurable? I can not see any configuration in the [`getMimeType`](https://github.com/getgrav/grav/blob/develop/system/src/Grav/Common/Utils.php#L264) function in [`Utils.php`](https://github.com/getgrav/grav/blob/develop/system/src/Grav/Common/Utils.php).

**Comment by flaviocopes** [comment]
This PR should address this issue: https://github.com/getgrav/grav/pull/370 instead of having mime types defined in getMimeType(), it picks the mime type from the media configuration. @rhukster can you review the change?
